### PR TITLE
LPS-136847 Add test to check that a site member cannot override a checkout process without permissions

### PR DESCRIPTION
**Pull request by DebSouza**
https://issues.liferay.com/browse/LPS-136847

### CODE REVIEW

**Comment by DebSouza** [comment]
ci:test:sf

**Comment by DebSouza** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/documentmanagement/documentsadministration/DMPermissions.testcase [hunk 1]
```diff
@@ -1452,6 +1506,18 @@ definition {
...
+
+		PermissionsInline.gotoDocumentsAndMediaDocumentPG(
+			dmDocumentTitle = "DM Document Title",
+			pageName = "Documents and Media Page");
+
+		for (var roleTitle : list "guest,site-member") {
+			Role.removePermissionPG(
+				permissionDefinitionKey = "CONTENT_PERMISSIONS_VIEW_CHECKBOX",
+				roleTitle = "${roleTitle}");
+		}
```
Comment (Jeremy-Ch) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/documentmanagement/documentsadministration/DMPermissions.testcase:1519]: Hi @DebSouza  I just want to confirm with you, why add these steps?

**Comment by DebSouza** [comment]
ci:test:sf

**Comment by Jeremy-Ch** [comment]
See: https://github.com/austinchiang/liferay-portal/pull/1204

**Pull request closed without merging**